hosted-ledgers/shop-with-hosted-ledger.js

```diff
@@ -0,0 +1,151 @@
+const http = require('http')
```

Comment:
You are doing many things manually here like setting up an HTTP server that returns the `Pay` header and other things. How about implementing such things in reusable modules and rely in the tutorials more on higher-level APIs like `ILP` etc.?

If this is a developer's first contact with ILP, it looks rather intimidating.

Reply:
that's true for dev docs where you don't care about what happens on the wire. here, we want to take off the lid and see actual headers being sent, and see their values getting calculated